inst/stan/functions/convolve.stan

```diff
@@ -12,6 +13,23 @@ vector convolve(vector x, vector y, int len) {
   return(convolution);
 }
 
+// convolve two vectors as a backwards dot product
+// y vector shoud be reversed
+// limited to the length of x and backwards looking for x indexes
+// designed for use convolve a case vector and a delay pmf
+vector convolve_dot_product(vector x, vector y, int len) {
+    int ylen = num_elements(y);
+    vector[len] z;
+    for (s in 1:len) {
+        z[s] = 1e-5 + // add a small value to ensure numerical stability
```

Comment:
Is there a specific need for this? I noticed in testing that removing this can change outcomes a fair bit.

Reply:
the `1e-5`? That was only there for numeric stability so may be able to remove. Not great if it has a large impact.

Reply:
I've dropped in [c63d92f](https://github.com/epiforecasts/EpiNow2/pull/343/commits/c63d92f36cb759097c04948cc638ddef7182f6da). Will play around with tomorrow.

Reply:
Played around with this a bit and it doesn't seem to make a major difference in terms of stability. Probably just a hang over of the bad old days code.